src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -50,6 +50,7 @@
 
 #include "AirflowNetwork/Solver.hpp"
 #include "AirflowNetwork/Properties.hpp"
+#include <EnergyPlus/AirflowNetworkBalanceManager.hh>
```

Comment:
@mitchute This is creating an `#include` cycle, which is not failing in this case but I'd generally try to avoid. It'd be better to pull the enums into this file than to include the manager header.

Reply:
OK, I can do that.